docs/site/BelongsTo-relation.md

```diff
@@ -212,14 +223,21 @@ export class Category extends Entity {
     super(data);
   }
 }
+
+export interface CategoryRelations {
+  categories?: CategoryWithRelations[];
+}
```

Comment:
I think this interface should include a navigation property for `parent` relation too, WDYT?

```ts
export interface CategoryRelations {
  categories?: CategoryWithRelations[];
  parent?: CategoryWithRelations;
}
```